### PR TITLE
Update KTLOSWorldBossLoot.json

### PR DESCRIPTION
**Pull request by shanemcdowell007**
We dont allow prio with world boss loot on EPGP. These are the current priorities based on who has been scouting so far. It is all decided on though by LC.

### CODE REVIEW

File: KTLOSWorldBossLoot.json [hunk 1]
```diff
@@ -95,123 +95,123 @@
...
 		"Fel Infused Leggings":{
-			"prio":["Gotosleep","EP/GP"],
+			"prio":["Gotosleep","LC"],
 			"gp:":"",
 			"wowID":""
 		},
 		"Empyrean Demolisher":{
-			"prio":["Drazic, but he has to sing the Star Spangled Banner for it","EP/GP"],
-			"gp:":"",
+			"prio":["Drazic","Chrixus","Mulligen","Wizzo","LC"],
```
Comment (dillmcpickle) [inline KTLOSWorldBossLoot.json:209]: delete mulligen and wizzo, its a dps item only

**Review by dillmcpickle — CHANGES_REQUESTED** [review]

File: KTLOSWorldBossLoot.json [hunk 2]
```diff
@@ -95,123 +95,123 @@
...
-			"prio":["EP/GP"],
-			"gp:":"",
+			"prio":["LC"],
+			"gp:":"1",
 			"wowID":""
 		},
 		"Mindtear Band":{
-			"prio":["EP/GP"],
-			"gp:":"",
+			"Give to mages":["LC"],
```
Comment (zhang165) [inline KTLOSWorldBossLoot.json:125]: keep as "prio"

File: KTLOSWorldBossLoot.json (hunk 1 above)
Comment (zhang165) [inline KTLOSWorldBossLoot.json:209]: +Dillmcpickle

File: KTLOSWorldBossLoot.json [hunk 3]
```diff
@@ -228,53 +228,53 @@
...
-			"prio":["EP/GP"],
-			"gp:":"",
+			"prio":["LC"],
+			"gp:":"5",
 			"wowID":""
 		},
 		"Unmelting Ice Girdle":{
-			"prio":["EP/GP"],
-			"gp:":"",
+			"prio":["Tank Prio for Naxx"],
```
Comment (zhang165) [inline KTLOSWorldBossLoot.json:251]: should be DPS war prio, tanks wear full mit

**Review by zhang165 — APPROVED** [review]

**Pull request merged**